ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -663,21 +663,17 @@ public static int get_permuted_index(int index, int listSize, Bytes32 seed) {
      */
 
     int indexRet = index;
-    byte[] byteTmp = new byte[1];
     byte[] powerOfTwoNumbers = {1, 2, 4, 8, 16, 32, 64, (byte) 128};
```

Comment:
While `(byte) 128` does overflow to -127 in Java, this is acceptable in all occurrences in `BeaconStateUtil` because it's only being used as a mask for bitwise operations. Just need to keep that in mind as we're using these powers of 2.